Userland/Libraries/LibWeb/CSS/StyleProperties.cpp

```diff
@@ -355,6 +355,22 @@ Optional<CSS::TextAlign> StyleProperties::text_align() const
     }
 }
 
+Optional<CSS::PointerEvents> StyleProperties::pointer_events() const
+{
+    auto value = property(CSS::PropertyID::PointerEvents);
+    if (!value.has_value() || !value.value()->is_identifier())
```

Comment:
Sorry I didn't catch this before, but `is_identifier()` isn't the right thing to call here. You can either change it to `has_identifier()`, or just remove the `|| !value.value()->is_identifier()` part altogether. (`to_identifier()` below will return `CSS::ValueID::Invalid` if there's no identifier, so it's always safe to call it.)

Reason for `has_identifier()` is that `auto` in particular is often parsed as a LengthStyleValue instead of an IdentifierStyleValue. `has_identifier()` works for `auto` in both of those cases. (This weirdness is on my list to sort out but I haven't got around to it yet.)

Reply:
I'm away from any computer until Friday but I'll get that changed as soon as I get back :)